miso-web/src/main/webapp/scripts/list_assay.js

```diff
@@ -6,37 +6,52 @@ ListTarget.assay = (function ($) {
     getUserManualUrl: function () {
       return Urls.external.userManual("requisitions", "assays");
     },
+    createUrl: function (config, projectId) {
+      throw new Error("Must be provided statically");
+    },
+    getQueryUrl: null,
     showNewOptionSop: true,
     createBulkActions: function (config, projectId) {
-      return !config.isAdmin
-        ? []
-        : [
-            {
-              name: "Copy",
-              action: function (items) {
-                if (items.length > 1) {
-                  Utils.showOkDialog("Error", ["Select an individual assay to copy"]);
-                  return;
-                }
-                window.location = Urls.ui.assays.create + "?" + $.param({ baseId: items[0].id });
-              },
+      if (config.projectId >= 0) {
```

Comment:
`>= 0` isn't ideal to check if the value is a number here because in JavaScript `null >= 0` is true. `projectId` will usually be undefined, so this will work currently, but there's potential for a confusing bug if it's ever set to null. Change to just check for a truthy value instead (`if (config.projectId) {...`)

Same below for static actions